web/src/transmission.js

```diff
@@ -622,7 +622,7 @@ export class Transmission extends EventTarget {
   static _isValidURL(string) {
     try {
       const url = new URL(string);
-      return url ? true : false;
+      return Boolean(url);
```

Comment:
Unnecessary. URL constructor always return URL for valid URL or `TypeError` exception is thrown when not valid URL. 

```
try {
    new URL(string);
    return true;
  } catch {
    return false;
 }
```

https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
4bf4797 Thanks!

Reply:
I see linter complain about no-new, maybe do this https://developer.mozilla.org/en-US/docs/Web/API/URL/parse_static as string type return value e.g. `return URL.parse(string) || '';` then maybe use that for loop constant anyway?

Reply:
Maybe we can use https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static too.

Or we do `return URL.parse(string) !== null`.

Reply:
I realized these are quite very new methods that might not prove high browser compatibility yet. You can do "polyfill" version of canParse by using isValidURL again then the return value in 'try' statement would be `return !!new URL(string);`

Reply:
`canParse()` is part of "Baseline 2023", so I guess by the time `4.1.0` is out, enough peoeple will be using a browser with support? 🤣

Reply:
December 2023. You may experience some pitchfork on this, don't worry that won't be me. XD

Reply:
Back to the starting point now.